### PR TITLE
[GSW-408] fix: Main Common Improve UI

### PR DESCRIPTION
**Pull request by khiemldk**

### CODE REVIEW

File: packages/web/src/components/common/wallet-connector-menu/WalletConnectorMenu.tsx [hunk 1]
```diff
@@ -92,17 +130,38 @@ const WalletConnectorMenu: React.FC<WalletConnectorMenuProps> = ({
...
             <IconButtonMaker
               copyClick={copyClick}
               openLinkClick={openLinkClick}
+              themeKey={themeKey}
+              copied={copied}
               onClickDisconnect={onClickDisconnect}
             />
           </MenuHeader>
-          <AmountInfoBox>{balanceText}</AmountInfoBox>
+          {account && account.chainId !== "dev.gnoswap" ? (
```
Comment (harryoh) [inline packages/web/src/components/common/wallet-connector-menu/WalletConnectorMenu.tsx:149]: this chainid should be get from environment variable `NEXT_PUBLIC_DEFAULT_CHAIN_ID`.

File: packages/web/src/components/common/wallet-connector-menu/WalletConnectorMenu.tsx [hunk 2]
```diff
@@ -14,22 +21,39 @@ import {
 import { formatAddress } from "@utils/string-utils";
 import ThemeModeContainer from "@containers/theme-mode-container/ThemeModeContainer";
 import { AccountModel } from "@models/account/account-model";
+import IconPolygon from "../icons/IconPolygon";
+import IconFailed from "../icons/IconFailed";
+
+const URL_REDIRECT =
+  "https://gnoscan.io/accounts/g14qvahvnnllzwl9ehn3mkph248uapsehwgfe4pt";
```
Comment (harryoh) [inline packages/web/src/components/common/wallet-connector-menu/WalletConnectorMenu.tsx:28]: What does this url mean?

Comment (khiemldk) [reply packages/web/src/components/common/wallet-connector-menu/WalletConnectorMenu.tsx:28]: Just Fake DATA (from log bug Kyungsu @harryoh )

File: packages/web/src/components/common/wallet-connector-menu/WalletConnectorMenu.tsx (hunk 1 above)
Comment (khiemldk) [reply packages/web/src/components/common/wallet-connector-menu/WalletConnectorMenu.tsx]: It's okay now

**Review by harryoh — APPROVED** [review]
👍

**Pull request merged**